analyzers/src/SonarAnalyzer.VisualBasic/SonarAnalyzer.VisualBasic.csproj

```diff
@@ -30,6 +30,18 @@
     <DefineConstants>TRACE;VB</DefineConstants>
   </PropertyGroup>
 
+  <Target Name="SignDlls" Condition="'$(SignAssembly)' == 'true' And '$(TargetFramework)' == 'NET46' " BeforeTargets="CopyBinaries">
+    <Message Importance="high" Text="Signing analyzer assemblies from $(OutputPath)" />
+    <Exec WorkingDirectory="$(OutputPath)" Command="&quot;$(SIGNTOOL_PATH)&quot; sign /fd SHA256 /f $(PFX_PATH) /p $(PFX_PASSWORD) /tr http://sha256timestamp.ws.symantec.com/sha256/timestamp SonarAnalyzer*.dll" />
+  </Target>
+
+  <Target Name="CopyBinaries" AfterTargets="Build" Condition=" '$(TargetFramework)' == 'NET46' ">
+    <ItemGroup>
+      <BinariesToCopy Include="$(OutputPath)\SonarAnalyzer.VisualBasic.dll" />
```

Comment:
Let's add these as well.
```
      <BinariesToCopy Include="$(OutputPath)\SonarAnalyzer.dll" />
      <BinariesToCopy Include="$(OutputPath)\Google.Protobuf.dll" />
```
Without that, cleaning CSharp project and rebuilding VB one will not play nicely with running ITs.